package.json

```diff
@@ -26,6 +26,9 @@
     "visual": "npx percy exec -- node snapshots.js",
     "test:a11y": "patternfly-a11y --config patternfly-a11y.config"
   },
+  "dependencies": {
+    "@patternfly/react-core": "^4.276.8"
```

Comment:
This should not be added here - the dependency is defined in the module/package.json - we can increase the version required there, though.